docs/workflows.md

```diff
@@ -1 +1,94 @@
-TODO
\ No newline at end of file
+# Workflows
+
+The purpose of this section of the Kn documentation is to list what we believe are common workflows or use-cases for the Knative CLI. Obviously, as we release the CLI in the wild and get users in production, we might discover novel workflows. As we do we will improve this document.
+
+## Basic (get your feet wet)
+
+In this basic worflow we show the CRUD (create, read, update, delete) operations on a service. We use a well known [simple Hello World service](https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-go) that reads the environment variable `TARGET` and prints it as output.
+
+1. Create a service from image in `default` namespace
+
+```bash
+$ kn service create hello --image gcr.io/knative-samples/helloworld-go --env TARGET=Knative
+```
+
+1. Curl service endpoint
+
+```bash
+curl '-sS' '-H' 'Host: hello.default.example.com' 'http://xxx.xx.xxx.xx:80'
+Hello Knative!
+```
+
+Where `http://xxx.xx.xxx.xx:80` is your Knative installation ingress.
+
+1. List service
+
+```bash
+$ kn service list
+$ hello
+```
+
+1. Update service
+
+```bash
+$ kn service update hello --env TARGET=Kn
+```
+
+The service's environment variable `TARGET` is now set to `Kn`
+
+1. Describe service
+
+```bash
+$ kn service describe hello
+apiVersion: knative.dev/v1alpha1
```

Comment:
The output of `kn service describe` will be always human readable, not yaml. Its `kn service get hello -o yaml` which returns a machine readable representation (with the same `-o` support as for kubectl as this is reused from Kubernete's cli-runtime).

Reply:
I'd agree with that. However, current implementation yielded YAML. I simply cut and pasted and edited to redact.

I'll try with latest code and if not ready I'll remove (for now).

Reply:
@rhuss no issue on this. Want to open it so we can see any feedback and I can address both.